readme.md

```diff
@@ -1,87 +1,130 @@
 # ovsrpro
 
-Build Overseer [dependencies](projects/README.md) by leveraging externpro
-(https://github.com/smanders/externpro).
+Build Overseer [dependencies](projects/README.md) by leveraging
+[externpro](https://github.com/smanders/externpro).
 
 ## Installing ovsrpro
 
 1. Download the installer for the release you need, for the your development
    environment (OS, compiler, etc). See
-   https://github.com/distributePro/ovsrpro/releases.
+   <https://github.com/distributePro/ovsrpro/releases>.
 1. Download the SHA256 checksum file for your release.
 1. Validate the checksum.
+   ```bash
+   sha256sum --check ovsrpro-18.10.1-gcc631-64-Linux.sh.sha256
+   ```
 1. Run the installer.
+   ```bash
+   sudo ./ovsrpro-18.10.1-gcc631-64-Linux.sh --prefix=/opt/extern/ --include-subdir
+   ```
+   The recommended location for installation is */opt/ovsrpro/*. We also
```

Comment:
If this is the recommended location, probably should change the `--prefix=/opt/extern/` in the example above to be `/opt/ovsrpro/`